rollup.config.ts

```diff
@@ -31,13 +43,7 @@ export default [
       },
     ],
     plugins: [
-      typescript({
-        useTsconfigDeclarationDir: true,
-        // https://github.com/ezolenko/rollup-plugin-typescript2/issues/105
-        objectHashIgnoreUnknownHack: true,
-      }),
-      commonjs(),
-      resolve(),
+      ...commonPlugins()
```

Comment:
nit: no need to spread, just `plugins === commonPlugins()`, unless you think leaving room for future node-specific plugins is useful.